index.html

```diff
@@ -70,7 +76,8 @@ <h1 class = "top-heading">I’m Sameed<br>Glad to see<br>you!</h1>
     <section class="middle-section">
 
         <article class="first-project">
-            <img src="./Images/1 tonic.png" alt="" class="work-image">
+            <img src="./Images/1 tonic.png" alt="" class="first-project-work-image">
+            <img src="./Images/DesktopCardWork1.png" alt="" class="desktop-card-work">
             <div class="details-container">
                 <div class="heading-and-highlights">
                     <h1 class="project-heading">Tonic</h1>
```

Comment:
- [ ] You did a good job styling your Works section :+1: :100:, but as a project requirement you should try to stick your design as much as possible with the Figma template. Below you can find some styling errors that I found

  - I have noticed that your cards are growing more than the ones of Figma :+1: 
  - Your title, as well as the other elements inside your project, seems to be not big enough like the Figma :+1: 

**Your design**

![Screenshot from 2023-01-26 06-35-38](https://user-images.githubusercontent.com/105117832/214827331-d35deb59-5ed6-4a23-90b3-d279374bf2c3.png)

**Figma**

![Screenshot from 2023-01-26 06-34-50](https://user-images.githubusercontent.com/105117832/214827396-c87bccb5-c903-4a60-b73f-42c33b312614.png)